packages/xsnap/lib/console-shim.js

```diff
@@ -2,10 +2,14 @@
 function tryPrint(...args) {
   try {
     // eslint-disable-next-line
-    print(...args);
+    print(...args.map(arg => typeof arg === 'symbol' ? arg.toString() : arg));
```

Comment:
Interesting. The C code for `print` already does `fprintf(stdout, "%s", xsToString(xsArg(i)));`.